rental.rb

```diff
@@ -1,8 +1,7 @@
 class Rental
+  attr_reader :book, :person
   attr_accessor :date
 
-  belongs_to :book, :person
-
   def initialize(date, person, book)
```

Comment:
- [ Required ] I think it would be nice if you refactor the `constructor` method so that the relationship between `Book`, `Person`, and `Rental` is set up correctly. Currently, the `person` and `book` objects don't have any association between them which causes the error below when listing the `Rentals` Please see the example below for a suggestion on how to set up the relationship below.

## Example code
```ruby
class Rental
  attr_reader :book, :person
  attr_accessor :date

  def initialize(date, person, book)
    @date = date
    @book = person
    @person = book

    person.rental << self
    book.rental << self
  end
end
```
## Error in project when listing rentals
![Screenshot from 2023-10-06 03-03-51](https://github.com/MenendezON/OOP-school-library/assets/16392046/2c964d48-c34a-49d8-8e42-db21962aac9c)